tests/vmi_configuration_test.go

```diff
@@ -2329,7 +2329,7 @@ var _ = Describe("Configurations", func() {
 			Expect(err).ToNot(HaveOccurred())
 			tests.WaitForSuccessfulVMIStart(vmi)
 
-			test_smbios := &cmdv1.SMBios{Family: "", Product: "", Manufacturer: ""}
+			test_smbios := &cmdv1.SMBios{}
```

Comment:
But an empty string is the default of a string, so this cannot be the reason of the problem. The CI still fails.

Reply:
Weird, that fixed it on my local kind setup... Really not sure what's happening here...

Reply:
I put together an ipv6 setup, reproduced and identified the problem.
`dmidecode` is not included by default in the Fedora image and for some reason we download it from a custom location instead of installing the package:
https://github.com/kubevirt/kubevirt/blob/master/tests/utils.go#L2091
At the time of download however, the VMI doesn't seem to have network connectivity, the download fails silently but `/usr/local/bin/dmidecode` is still created as an empty executable file, which happily runs and returns 0 with no output.
The latest Fedora upgrade might have changed something that we now need to fix.

Reply:
So you are saying this test always failed, but only now (after fixing the expecter) we see it?
I know there are other tests, which fetch the guest agent in the ipv6 lane and it works.

Can you revert back this line in the meantime? I can try to debug the network connectivity on the ipv6 lane.

Reply:
Done

Reply:
Ok, I see the problem.
These tests use `NewRandomFedoraVMIWithDmidecode` which is broken on the IPv6 lane because the IP addresses are defined only at the console "login" stage (`LoggedInFedoraExpecter`).

Now that we have cloud-init network-data support and when #3433 will get in, it should be a simple fix.

A quick workaround could be to install the package from the console at which time an IP should be defined already.
If that does not fit and as I do not see any other quick fix/workaround here, maybe you can skip these tests from the IPv6 lane and open an issue that tracks it (so we will fix it).

Reply:
Let's not abuse the expecter more than it already is, especially since a better solution is almost ready.
I followed your suggestion, created an issue (https://github.com/kubevirt/kubevirt/issues/3901) and added a commit with the skips.